infrastructure-formula/infrastructure/salt/files/srv/reactor/update_fileserver.sls

```diff
@@ -0,0 +1,17 @@
+{%- set deploy_password = salt['pillar.get']('infrastructure:salt:reactor:update_fileserver_deploy_password', '') -%}
+{%- raw -%}
+{%- if data.data == "{% endraw %}{{ deploy_password }}{% raw %}" -%}
+{%- endraw %}
+update_fileserver:
+  runner.fileserver.update: []
+  runner.git_pillar.update: []
+{%- if grains['domain'] == 'infra.opensuse.org' %} {#- to-do: find a common solution #}
+update_fileserver_ng:
+  local.state.apply:
+    - tgt: minnie.infra.opensuse.org
```

Comment:
Hehe, "you shall not have a saltmaster besides me" ;-))

Maybe use a pillar in `tgt:`, and also check that pillar for the if condition above?

Reply:
Right, I remember `grains['master']` not yielding the right result in this context. I wonder if pillar would contain the right result for non-global top includes. Would need to test this. Currently it is only used in openSUSE, but it could still be useful to configure it using the pillar.

Reply:
Actually, `update_fileserver_deploy_password` is coming from the pillar as well. So adding it there sounds like a good idea.

Reply:
What about now ?